Misc/NEWS.d/next/macOS/2018-12-21-18-44-30.bpo-35555.M58_K3.rst

```diff
@@ -0,0 +1 @@
+Gray out Code Context menu entry when it's not applicable.
```

Comment:
Putting this in the macOS instead of IDLE section confused Ned into thinking this was a Mac-only change.  We should check each other's blurb directories.  It is too easy to get the wrong one.